<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLEditText</class>
 <widget class="QDialog" name="WLEditText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit text</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="5" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="gbFind">
     <property name="title">
      <string>Find</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbBackFind">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditFind"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="WLScriptFuncComboBox" name="cbFunction"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WLScriptFuncComboBox</class>
   <extends>QComboBox</extends>
   <header>wlscriptfunccombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WLEditText</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WLEditText</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
